src/components/App/index.js

```diff
@@ -45,6 +49,9 @@ class App extends Component {
   }
 
   renderBody() {
+    if (!this.props.requests) {
+      this.props.fetchPrs(this.props.oauthToken, this.props.loggedInUser);
```

Comment:
Why not leverage a lifecycle method here? You could put this in [componentWillMount](https://developmentarc.gitbooks.io/react-indepth/content/life_cycle/birth/premounting_with_componentwillmount.html)